James/book_class.py

```diff
@@ -0,0 +1,19 @@
+class Booklist():
+	def __init__(self):
+		self.books = []
+
+
+	def add(self, title, author):
```

Comment:
When you run these files, there is an error code with this function. Were you able to see this?

The add function should create an empty dictionary 'book' , set dictionary keys 'title' and 'author' to the method attributes, then append as the last step

def add(self, title, author)
   book = {}
   book['title'] = title
   book['author'] = author

   self.books.append(book)